diff -urN rfremix-config-15.orig/rfremixconf.init rfremix-config-15/rfremixconf.init
--- rfremix-config-15.orig/rfremixconf.init	2011-04-23 01:47:16.632271778 +0400
+++ rfremix-config-15/rfremixconf.init	2011-04-23 01:47:33.424180923 +0400
@@ -81,6 +81,7 @@
 	if [ -f /etc/smolt/config.py ]; then
 	    sed -i 's!OS = os_detect.get_os_info!#OS = os_detect.get_os_info!g' /etc/smolt/config.py
 	    sed -i 's!#    OS = file!OS = file!g' /etc/smolt/config.py
+	    sed -i 's!distro-release!rfremix-release!g' /etc/smolt/config.py
 	fi
 
 	# end of rules
